src/actions/CryptoExchangeActions.js

```diff
@@ -577,6 +452,7 @@ export const selectWalletForExchange = (walletId: string, currencyCode: string)
 }
 
 export const exchangeTimerExpired = () => (dispatch: Dispatch, getState: GetState) => {
+  Actions[Constants.EXCHANGE_QUOTE_PROCESSING_SCENE]()
```

Comment:
What does this do? The timer expires regularly every few seconds. Why are you trying to route to the quoteProcessing scene without user input?

Reply:
Actually I am glad you caught this, but it is not what you think. Shape shift quotes have an expiration date. When the time expires, the timer component calls this function. I am taking the quote they previously requested, resubmitting it to get a new quote so they can then execute it if they are still on the page. The redirect keeps the user from trying to accept an expired quote. Once the quote refreshes, then they are back on the confirmation screen